Resources/Private/Fusion/Prototypes/Radio.fusion

```diff
@@ -0,0 +1,13 @@
+prototype(Neos.Fusion.Form:Radio)  < prototype(Neos.Fusion.Form:Field) {
+    attributes = Neos.Fusion:DataStructure
+    value = ''
+    renderer = afx`
+        <input
+            {...props.attributes}
+            type="radio"
+            name={field.name}
+            value={props.value}
+            checked={field.value == this.value}
```

Comment:
Hey,

if I understand the code correctly, you need to set "field.value" for e.g. textboxes etc; but for radio boxes you need to set "value" directly.

Correct? If so, this wouldn't be good API-wise -- and I'd say having an external API which is stable and makes sense is crucial.